packages/koa-bolt/src/index.spec.ts

```diff
@@ -0,0 +1,95 @@
+import Router from '@koa/router';
+import { events, ServerlessTester } from '@slack-wrench/fixtures';
+import { App, ExpressReceiver } from '@slack/bolt';
+import Koa, { Middleware } from 'koa';
+
+import koaBolt from './index';
+
+const identityRoute = (path: string): Middleware<any, any> => {
+  const route = new Router();
+
+  route.get(path, ctx => {
+    ctx.body = path;
+  });
+
+  return route.middleware();
+};
```

Comment:
can write as 

```ts
const identityRoute = (path: string): Middleware<any, any> =>
  new Router()
    .get(path, ctx => {
      ctx.body = path;
    })
    .middleware();
```
I think?

Reply:
That does indeed work, that's a lovely thing I hadn't realized worked.